packages/wdio-cli/src/config.js

```diff
@@ -290,7 +298,19 @@ export const QUESTIONNAIRE = [{
     name: 'services',
     message: 'Do you want to add a service to your test setup?',
     choices: SUPPORTED_SERVICES,
-    filter: filterPackageName('service')
+    default: SUPPORTED_SERVICES.filter(service => service.includes('wdio-chromedriver-service')),
```

Comment:
Why not just?
```suggestion
    default: 'wdio-chromedriver-service',
```

Reply:
@christian-bromann that's what I've tried first :)

First of all it should be an array for checkboxes, secondly I have to specify full name which is `' chromedriver - https://www.npmjs.com/package/wdio-chromedriver-service'`

I can change `SUPPORTED_SERVICES.filter(service => service.includes('wdio-chromedriver-service'))` to `' chromedriver - https://www.npmjs.com/package/wdio-chromedriver-service'`

Same for reporters

Reply:
ah got it, nah this is fine then